packages/babel-helpers/src/helpers.js

```diff
@@ -943,9 +943,7 @@ helpers.applyDecoratedDescriptor = () => template.program.ast`
         }
 
         if (desc.initializer === void 0){
-            // This is a hack to avoid this being processed by 'transform-runtime'.
-            // See issue #9.
-            Object['define' + 'Property'](target, property, desc);
+            Object.defineProperty(target, property, desc);
```

Comment:
@loganfsmyth I hope that it should not cause a problem now?

Reply:
(related https://github.com/loganfsmyth/babel-plugin-transform-decorators-legacy/issues/9)

Reply:
Yeah I think this is fine now.